Cargo.toml

```diff
@@ -26,7 +26,7 @@ atty = "0.2"
 clap = { version = "3", features = ["std", "derive", "suggestions", "color"], default-features = false }
 env_logger = { version = "0.9", optional = true }
 log = "0.4"
-reqwest = { version = "0.11.3", features = ["blocking", "rustls-tls", "rustls-tls-native-roots"], default-features = false }
+reqwest = { version = "0.11.3", features = ["blocking"], default-features = false }
```

Comment:
Why did the `rustls-tls` feature get removed too?

Reply:
https://docs.rs/reqwest/latest/reqwest/index.html#optional-features

> rustls-tls: Enables TLS functionality provided by rustls. Equivalent to rustls-tls-webpki-roots.